CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Not released
 
+- PieWidgetUI: Refactor & New CentralText component [#779](https://github.com/CartoDB/carto-react/pull/779)
+- Pie widget: New ChartLegend component from scratch [#777](https://github.com/CartoDB/carto-react/pull/777)
+- Pie Widget: Add maxNumber of elements + a sorted by size desc by default [#774](https://github.com/CartoDB/carto-react/pull/774)
+- Pie Widget: Add number of selected categories + clear button [#771](https://github.com/CartoDB/carto-react/pull/771)
```

Comment:
@aaranadev, @menusal we want to land this improved PieWidget very soon. We'll ping you once we have the release, so you can test in PS projs